syncd/RequestShutdown.h

```diff
@@ -9,6 +9,18 @@
 
 #define SYNCD_NOTIFICATION_CHANNEL_RESTARTQUERY     "RESTARTQUERY"
 
+/**
+ * @brief Notification channel 'restartQuery' per DB scope
+ *
+ * In https://redis.io/docs/manual/pubsub/, it says:
+ * "Pub/Sub has no relation to the key space. It was made to not interfere with
+ * it on any level, including database numbers."
+ */
+#define SYNCD_NOTIFICATION_CHANNEL_RESTARTQUERY_PER_DB(dbName) \
+    ((dbName) == "ASIC_DB" ? \
```

Comment:
Also this compare can be decieving since if it's a stdsteing you get what you want if this is literal then not so much